lib/flatpak-remote.h

```diff
@@ -25,6 +25,15 @@
 #ifndef __FLATPAK_REMOTE_H__
 #define __FLATPAK_REMOTE_H__
 
+#ifdef FLATPAK_ENABLE_P2P
+typedef enum
```

Comment:
Please don't make the libflatpak public headers ifdef:ed.
It should be completely safe to always have these available, as they are purely declarative and there is no way to create such remotes anyway. That means even if they have to change later (which i doubt) that doesn't really matter.